### ISSUE TITLE
Let us clean up our analytics

### ISSUE THREAD

**Issue by heytherewill** [toggl/mobileapp#5635]
## 🚧 Refactor Proposal
As discussed in Buenos Aires, there are quite a lot of ugly analytics code that can be removed if it's no longer needed. With that in mind, @amulware agreed to review our telemetry systems and let us know what things can be discarded.

## Dependencies
Before or after #5634, but not at the same time.

## Technical aspects
Paul finds what needs to be 🔥, but we shouldn't remove the events from our code for two reasons:

- We might wanna use them again, and in that case it'd be good to use the same event name
- We might need to create an event with a similar name, in that case we shouldn't use the same name.

Marking the events as `[Obsolete("Event no longer in use", true)]` should be enough, but everyone is free to argue otherwise.
